tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_provision.c

```diff
@@ -14,20 +16,56 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 
+typedef unsigned int uint;
+
 /*
  * Provision layer tests:
  *   Tests both the provisioner and device role in various scenarios.
  */
 
 #define PROV_MULTI_COUNT 3
+#define WAIT_TIME 60 /*seconds*/
 
 enum test_flags {
 	IS_PROVISIONER,
 
 	TEST_FLAGS,
 };
 
+static uint8_t static_key1[] = {0x6E, 0x6F, 0x72, 0x64, 0x69, 0x63, 0x5F,
+		0x65, 0x78, 0x61, 0x6D, 0x70, 0x6C, 0x65, 0x5F, 0x31};
+static uint8_t static_key2[] = {0x6E, 0x6F, 0x72, 0x64, 0x69, 0x63, 0x5F};
+
+static struct {
+	const uint8_t *static_val;
+	uint8_t        static_val_len;
+	uint8_t        output_size;
+	uint16_t       output_actions;
+	uint8_t        input_size;
+	uint16_t       input_actions;
+} oob_auth_test_vector[] = {
+	{static_key1, sizeof(static_key1), 0, 0, 0, 0},
+	{static_key2, sizeof(static_key2), 0, 0, 0, 0},
+	{NULL, 0, 3, BT_MESH_BLINK, 0, 0},
+	{NULL, 0, 5, BT_MESH_BEEP, 0, 0},
+	{NULL, 0, 6, BT_MESH_VIBRATE, 0, 0},
+	{NULL, 0, 7, BT_MESH_DISPLAY_NUMBER, 0, 0},
+	{NULL, 0, 8, BT_MESH_DISPLAY_STRING, 0, 0},
+	{NULL, 0, 0, 0, 4, BT_MESH_PUSH},
+	{NULL, 0, 0, 0, 5, BT_MESH_TWIST},
+	{NULL, 0, 0, 0, 8, BT_MESH_ENTER_NUMBER},
+	{NULL, 0, 0, 0, 7, BT_MESH_ENTER_STRING},
+};
+
 static ATOMIC_DEFINE(flags, TEST_FLAGS);
+
+extern void bs_clean_back_channels(void);
```

Comment:
Why not just include `bs_pc_backchannel.h`?

Reply:
no path to the header

Reply:
in principle, it is a bad idea to do source code dependable on the third part repository

Reply:
That seems strange as the path is added here: https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/posix/nrf52_bsim/CMakeLists.txt#L37
And here `bs_cmd_line.h` is included without any issues: https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_persistence.c#L14

Reply:
I see. It works only with `system` header including. I'll add it but it looks weird